src/serac/physics/nonlinear_solid.hpp

```diff
@@ -80,15 +80,15 @@ class NonlinearSolid : public BasePhysics {
    * @param[in] mesh The MFEM parallel mesh to solve on
    * @param[in] solver The system solver parameters
    */
-  NonlinearSolid(int order, std::shared_ptr<mfem::ParMesh> mesh, const SolverParameters& params);
+  NonlinearSolid(int order, std::shared_ptr<mfem::ParMesh> mesh, const SolverOptions& options);
 
   /**
    * @brief Construct a new Nonlinear Solid Solver object
    *
    * @param[in] mesh The MFEM parallel mesh to solve on
    * @param[in] info The solver information parsed from the input file
    */
-  NonlinearSolid(std::shared_ptr<mfem::ParMesh> mesh, const InputInfo& info);
+  NonlinearSolid(std::shared_ptr<mfem::ParMesh> mesh, const InputOptions& info);
```

Comment:
sorry `info` to `options`

Reply:
ARG!

Reply:
Ha! I already did that but I hadn't pushed yet!